scipy/signal/filter_design.py

```diff
@@ -161,6 +173,94 @@ def freqs(b, a, worN=None, plot=None):
     return w, h
 
 
+def freqs_zpk(z, p, k, worN=None, plot=None):
```

Comment:
Also does this function need to be public if `freqresp` provides a better interface to do the same thing?